### PR TITLE
fix: replaced 'fs' with 'fs-extra' completely

### PR DESCRIPTION
**Pull request by saadjutt01**
## Intent

Remove `fs` package 
> This package name is not currently in use, but was formerly occupied by another package. To avoid malicious use, npm is hanging on to the package name, but loosely, and we'll probably give it to you if you want it.

## Implementation

Replaced `fs` with already using `fs-extra`

## Checks

- [x] Code is formatted correctly (`npm run lint:fix`).
- [ ] Any new functionality has been unit tested.
- [x] All unit tests are passing (`npm test`).
- [ ] All CI checks are green.
- [ ] JSDoc comments have been added or updated.
- [ ] Reviewer is assigned.

### CODE REVIEW

**Review by allanbowe — DISMISSED** [review]

**Review by YuryShkoda — CHANGES_REQUESTED** [review]
Please move this logic to @sasjs/utils

File: src/cli.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ import {
   ReturnCode,
   test
 } from './main'
-import { fileExists } from './utils/file'
+import { fileExists } from '@sasjs/utils/file'
```
Comment (YuryShkoda) [inline src/cli.ts:26]: we should import utils from `@sasjs/utils`

**Review by YuryShkoda — CHANGES_REQUESTED** [review]

File: src/cli.ts (hunk 1 above)
Comment (saadjutt01) [reply src/cli.ts]: done

File: src/utils/file.js [hunk 2]
```diff
@@ -38,159 +36,10 @@ export async function createFolderStructure(folder, parentFolderName = '.') {
...
-    fs.writeFile(fileName, content, function (error) {
-      if (error) {
-        process.logger?.error(`Error creating file ${fileName}`)
-        return reject(error)
-      }
-
-      resolve(content)
-    })
-  })
+export async function listIniFilesInFolder(folderName) {
```
Comment (YuryShkoda) [inline src/utils/file.js:39]: this utility should be in `@sasjs/utils`

Comment (saadjutt01) [reply src/utils/file.js]: done

**Pull request merged**